assembly/assembly-wsmaster-war/pom.xml

```diff
@@ -87,6 +87,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.che.core</groupId>
+            <artifactId>che-core-api-auth-github</artifactId>
+            <version>7.0.0-beta-3.0</version>
```

Comment:
dep version should be removed

Reply:
Right, good catch; it was a copy paste from maven

Fixed.